examples/fatigue/static_arm_with_fatigue.py

```diff
@@ -0,0 +1,161 @@
+"""
+This is a basic example on how to use biorbd model driven by muscle to perform an optimal reaching task.
+Fatigue is applied on muscles. The arms must reach a marker placed upward in front while minimizing
+the muscles activity and fatigue.
+Please note that using show_meshes=True in the animator may be long due to the creation of a huge CasADi graph of the
+mesh points.
+"""
+
+import biorbd_casadi as biorbd
+
+from bioptim.misc.enums import Fatigue
```

Comment:
This should not be imported like this. It should be imported via `from bioptim` if it does not appear, that means it has to be added in __init__.py

Reply:
That said, it does not seem you are using it..